src/NuGet.Core/NuGet.Commands/CLILanguageOverrider.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NuGet.Common;
+
+namespace NuGet.Commands
+{
+    /// <summary>Set a CLI culture using environment variables</summary>
+    // This type is public because it's the only way to call it from NuGet.CommandLine.XPlat (dotnet.exe) and NuGet.CommandLine (nuget.exe) projects
+    // without introducing a new assembly
```

Comment:
```suggestion
    /// <summary>
    /// Set a CLI culture using environment variables.
    /// This type is public because it's the only way to call it from NuGet.CommandLine.XPlat (dotnet.exe) and NuGet.CommandLine (nuget.exe) projects 
    /// without introducing a new assembly.
    /// </summary>
```

Reply:
Thanks! Added as a remarks section since it's a implementation detail.